index.js

```diff
@@ -8,6 +8,9 @@ require("./src/CSVEncoding.js");
 require("./src/JSONEncoding.js");
 require("./src/forgivingJoin.js");
 require("./src/dataPipeline.js");
+require("./src/processPipes.js");
+require("./src/ioPipes.js");
+require("./src/processPipes.js");
```

Comment:
"./src/processPipes.js" is included twice at line 11 & 13. One of them can be removed.